src/scheduler/fifo.c

```diff
@@ -1383,12 +1381,8 @@ run_update_resresv(status *policy, int pbs_sd, server_info *sinfo,
 
 	pbs_errno = PBSE_NONE;
 	if (resresv->is_job && resresv->job->is_suspended) {
-		if (pbs_sd != SIMULATE_SD) {
-			pbsrc = pbs_sigjob(pbs_sd, resresv->name, "resume", NULL);
-			if (!pbsrc)
-				ret = 1;
-		}
-		else 
+		pbsrc = pbs_sigjob(pbs_sd, resresv->name, "resume", NULL);
```

Comment:
when in simulation what happens in pbs_sigjob? wouldn't it log any error message because pbs_sd passed was invalid?